iroha_data_model/src/lib.rs

```diff
@@ -301,6 +302,28 @@ pub trait Identifiable: Debug + Clone {
     type Id: Into<IdBox> + Debug + Clone + Eq + Ord;
 }
 
+/// Limits of length of the identifiers (e.g. in [`Domain`], [`NewAccount`], [`AssetDefinition`])
+#[derive(Debug, Clone, Copy, Decode, Encode, Serialize, Deserialize)]
+pub struct LengthLimits {
+    /// Minimal length (inclusive).
+    min_len: u32,
+    /// Maximal length (inclusive).
+    max_len: u32,
```

Comment:
```suggestion
    /// Minimal length (inclusive).
    min: u32,
    /// Maximal length (inclusive).
    max: u32,
```

Reply:
No need to duplicate the name as it is already in `LengthLimits`

Reply:
Renamed